project/api/models/account.py

```diff
@@ -273,16 +273,17 @@ def get_voted_bills(self):
 
         for activity in activities.iterator():
             if activity.is_negative_vote:
-                self._add_linked_civis(supported_bills, activity)
+                if activity.civi.c_type == 'cause':
+                    supported_bills += activity.civi.linked_bills.all()
+                elif activity.civi.c_type == 'solution':
+                    opposed_bills += activity.civi.linked_bills.all()
             elif activity.is_positive_vote:
-                self._add_linked_civis(opposed_bills, activity)
+                if activity.civi.c_type == 'cause':
```

Comment:
Add a comment here describing why causes are opposed and solutions are supported, to make it easier for future developers to understand.